### PR TITLE
Bug 24

### PR DESCRIPTION
**Pull request by filipgorczynski**

### CODE REVIEW

**Comment by Yog** [comment]
From mine perspective looks alright

File: volontulo/urls.py [hunk 1]
```diff
@@ -41,6 +41,11 @@
         name='offer_form'
     ),
     url(
+        r'^offers/form/(?P<organization_id>[0-9]+)/(?P<offer_id>[0-9]+)$',
```
Comment (magul) [inline volontulo/urls.py:38]: Why we need to have organization-id when we want to edit particular offer fully identified by offer-id?

Comment (magul) [inline volontulo/urls.py:38]: Ok, I have found why, but it looks, that it will not wokr if You want to create new offer (if You don't have offer-id).

Comment (filipgorczynski) [inline volontulo/urls.py:38]: If I have no offer_id in URL there is offers/form/organization_id URL used instead.

**Pull request merged**